### PR TITLE
docs: Update troubleshooting to include debug logging option

### PR DESCRIPTION
**Pull request by rothgar**
Fixes # 

**Description**
Added a general troubleshooting section and documented how to enable debug logging.

Automatic md formatting was also applied which adds lines before and after code blocks and changes * bullets to - (I think they're more portable for md renderers)

**How was this change tested?**

* locally

**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note
Added docs for debug logging
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/troubleshooting.md [hunk 1]
```diff
@@ -6,14 +6,39 @@ description: >
   Troubleshoot Karpenter problems
 ---
 
+## Controller
+
+### Enable debug logging
```
Comment (jonathan-innis) [inline website/content/en/preview/troubleshooting.md:11]: Can we include info on how to do this with helm as well?

Comment (rothgar) [reply website/content/en/preview/troubleshooting.md:11]: Do you know what the differences is between `logLevel` which is used in the zap-logger-config and `controller.logLevel` which is used separately in the helm template?

https://github.com/aws/karpenter/blob/main/charts/karpenter/templates/configmap-logging.yaml#L39

File: website/content/en/preview/troubleshooting.md [hunk 2]
```diff
@@ -6,14 +6,49 @@ description: >
...
+      "level": debug",
+      development": false,
+...
+```
+
+Update the zap-logger-config "level" and restart the Karpenter pod(s) to enable debug logging.
+
+#### Debug logging via Helm
+
+You can enable debug logging during installation with helm by setting the option `contreller.logLevel`.
```
Comment (jonathan-innis) [inline website/content/en/preview/troubleshooting.md:32]:
```suggestion
You can enable debug logging during installation with helm by setting the option `logLevel`.
```

File: website/content/en/preview/troubleshooting.md (hunk 1 above)
Comment (jonathan-innis) [reply website/content/en/preview/troubleshooting.md:11]: `controller.logLevel` is an artifact of an old release and needs to be pulled out and removed from the values.yaml. I'll file a separate issue to cleanup the helm values

**Review by jonathan-innis — DISMISSED** [review]
I think this just needs a rebase, but otherwise, LGTM!

**Review by ellistarn — DISMISSED** [review]

**Comment by rothgar** [comment]
I tried to use the github UI to rebase. That was a mistake. It should be ok now.

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀

**Pull request merged**